File: day058/classwork/cw01.py
def number(a,b):
    if a % 2 == 0:
        a = a // 2
    else:
        a = a + 1

    return a - b

print(number(10,24))
print(number(56,45))
#return არ გამოაქვს შედეგი ტერმინალში print გამოაქვს

File: day146/classwork/cw01.js
// მოცემულია ობიექტტი მოცემული ობიექტიდან დესტრუქცია გაუკეთე name და city ს დანარჩენი კი გამოიტანე სპრედ ოპერატორის დახმარებით
const store = {
  name: "MegaShop",
  address: {
    city: "Batumi",
    location: {
      street: "Chavchavadze",
      number: 10
    }
  },
  products: [
    {
      name: "Laptop",
      price: 2500
    },
    {
      name: "Phone",
      price: 1200
    }
  ]
} 

const{name,address:{city},...rest} = store
console.log(name)
console.log(city)
console.log(rest)

